spk/dnscrypt-proxy/src/service-setup.sh

```diff
@@ -45,6 +51,10 @@ forward_dns_dhcpd () {
 service_prestart () {
     echo "service_preinst ${SYNOPKG_PKG_STATUS}" >> "${INST_LOG}"
 
+    # Install daily cron job, to update the block list
+    echo "3       0       *       *       *       root    /var/packages/dnscrypt-proxy/target/var/update-blocklist.sh" >> /etc/crontab
```

Comment:
Please move application specific cron job from system "crontab". I have tested, it works perfectly when declared in a specific file in `/etc/cron.d`. So I propose to write to job into `/etc/cron.d/dnscrypt-proxy-update-blocklist`

Reply:
I tested it putting the file on my router but unfortunately this will not work since busybox cron does not use 'cron.d' folder:

https://git.busybox.net/busybox/tree/miscutils/crond.c

http://lists.busybox.net/pipermail/busybox/2016-January/083783.html

Reply:
Sorry I am not sure to understand. Do you mean that SRM system does not run same cron service like DSM, but busybox cron service?

Reply:
Yes, the RSM uses most utilities from busybox. Normally it doesn’t matter but in this case it does.

Reply:
FYI the shell is from busybox too. (ash)

Reply:
I know it may look overkill, but would it be possible to conditionally add cron entry depending on DSM or SRM system? based on available environment variables: https://originhelp.synology.com/developer-guide/synology_package/script_env_var.html
I would really appreciate the effort.

Reply:
Done